docs/src/apps/backwards-compatibility.md

```diff
@@ -0,0 +1,141 @@
+that were enabled in the previous `MAJOR` version.
+
+#### Minor Releases (1.x.0)
+
+New features and proposal implementations are added to _new_ `MINOR` version
+releases (e.g. 1.4.0) and are first run on Solana's Tour de SOL testnet cluster. While running
+on the testnet, `MINOR` versions are considered to be in the `beta` release channel. After
+those changes have been patched as needed and proven to be reliable, the `MINOR` version will
+be upgraded to the `stable` release channel and deployed to the Mainnet Beta cluster.
+
+#### Patch Releases (1.0.x)
+
+Low risk features, non-breaking changes, and security and bug fixes are shipped as part
+of `PATCH` version releases (e.g. 1.0.11). Patches may be applied to both `beta` and `stable`
+release channels.
+
+### RPC API
+
+Patch releases:
+- Bug fixes
+- Security fixes
+- Endpoint / feature deprecation
+
+Minor releases:
+- New RPC endpoints and features
+
+Major releases:
+- Removal of deprecated features
+
+### Rust Crates
+
+* [`solana-sdk`](https://docs.rs/solana-sdk/) - Rust SDK for creating transactions and parsing account state
+* [`solana-program`](https://docs.rs/solana-program/) - Rust SDK for writing programs
+* [`solana-client`](https://docs.rs/solana-client/) - Rust client for connecting to RPC API
+* [`solana-cli-config`](https://docs.rs/solana-cli-config/) - Rust client for managing Solana CLI config files
+
+Patch releases:
+- Bug fixes
+- Security fixes
+- Performance improvements
+
+Minor releases:
+- New APIs
+
+Major releases
+- Removal of deprecated APIs
+- Backwards incompatible behavior changes
+
+### CLI Tools
+
+Patch releases:
+- Bug and security fixes
+- Performance improvements
+- Subcommand / argument deprecation
+
+Minor releases:
+- New subcommands
+
+Major releases:
+- Switch to new RPC API endpoints / configuration introduced in the previous major version.
+- Removal of deprecated features
+
+### Runtime Features
+
+New Solana runtime features are feature-switched and manually activated. Runtime features
+include: the introduction of new native programs, sysvars, and syscalls; and changes to
+their behavior. Feature activation is cluster agnostic, allowing confidence to be built on
+Testnet before activation on Mainnet-beta.
+
+The release process is as follows:
+
+1. New runtime feature is included in a new release, deactivated by default
+2. Once sufficient staked validators upgrade to the new release, the runtime feature switch
+  is activated manually with an instruction
+3. The feature takes effect at the beginning of the next epoch
+
+### Infrastructure Changes
+
+#### Public API Nodes
+
+Solana provides publicly available RPC API nodes for all developers to use. The Solana team
+will make their best effort to communicate any changes to the host, port, rate-limiting behavior,
+availability, etc. However, we recommend that developers rely on their own validator nodes to
```

Comment:
To this point, should we include rate-limiting info on docs.solana.com/clusters with the RPC URL?